assets/js/admin/components/promos/index.js

```diff
@@ -9,7 +9,7 @@ jQuery( document ).ready( function( $ ) {
 		const topOfPageNoticeEl = topOfPageNotice.detach();
 
 		$( '#wpbody-content' ).prepend( topOfPageNoticeEl );
-		topOfPageNotice.delay( 1000 ).slideDown();
+		topOfPageNotice.show();
```

Comment:
I wonder if this could still be a slideDown, just without the delay. The show feels abrupt. If we are just going to show it right away, it almost seems like we could consider just rendering with PHP instead of dipping into JS.

Reply:
@robincornett Fixed this up.